plugins/catalog-backend/src/schema/openapi.yaml

```diff
@@ -850,6 +850,10 @@ paths:
                   type: array
                   items:
                     type: string
+                filter:
```

Comment:
Think it could be more useful the use the `filter` `ref` instead? https://github.com/backstage/backstage/blob/b8aba5c540948109221793bc613c91c29047572b/plugins/catalog-backend/src/schema/openapi.yaml#L892C9-L892C49

Reply:
Actually cannot use that because the $ref says it's in the query while this is part of the request body